events/wc_druidic_events.txt

```diff
@@ -69,7 +98,10 @@ character_event = {
 						is_druid_class_trigger = yes
 						is_loa_priest_class_trigger = yes
 					}
-					is_5_6_magic_class_trigger = yes
```

Comment:
It's here to check level. Class is checked above. Plus, I wouldn't make it for druids and mages only. I can imagine shaman or loa priest doing it.

Reply:
It was always intended to be for all 4. When there were 2 event chains one was meant for mages, and the other for the rest. I confused the mage and magic triggers when consolidating. Fixed.